include/ortx_tokenizer.h

```diff
@@ -60,6 +60,18 @@ extern "C" {
  */
 extError_t ORTX_API_CALL OrtxCreateTokenizer(OrtxTokenizer** tokenizer, const char* tokenizer_path);
 
+/** \brief Create a tokenizer object with the specified tokenizer path and options such as add_special_tokens for tokenizing, or skip_special_tokens for detokenizing.
+ *
+ * \param tokenizer Pointer to store the created tokenizer object
+ * \param tokenizer_path The path to the tokenizer directory, which is utf-8 encoded
+ * \param option_keys Array of option keys
+ * \param option_values Array of option values (same length as keys)
+ * \param num_options Number of options
+ * \return Error code indicating the success or failure of the operation
+ */
+extError_t ORTX_API_CALL OrtxCreateTokenizerWithOptions(OrtxTokenizer** tokenizer, const char* tokenizer_path, const char* option_keys[], const char* option_values[], size_t num_options);
```

Comment:
Could we list all the valid tokenizer options available. Please also specify the expected values for each options if possible.

Reply:
Done!